### DISCUSSION TITLE
Parse copy-pasted secrets in the dashboard

### DISCUSSION THREAD

**Discussion by vmatsiiako** [Infisical/infisical#5704] [Feature Request]
### Feature description
If a user pastes `SECRET_X=VALUE_Y` into the dashabord, we should immediately split the key and the values into two input fields. 

### Why would it be useful?
This a QoL feature and it would be dramatically improve the user experience for many users who paste the environment variables / secrets into the dashaboard.

**Comment by DarkPhoenix2704** [comment]
Hey @mv-turtle. Can I take this issue?

**Comment by vmatsiiako** [comment]
@DarkPhoenix2704 yes, please! This would very helpful!

Are you already in our slack? If not, you should join :)

Link: https://join.slack.com/t/infisical-users/shared_invite/zt-1kdbk07ro-RtoyEt_9E~fyzGo_xQYP6g

**Comment by alceil** [comment]
Hey @vmatsiiako Can I take a dab at this?

**Comment by vmatsiiako** [comment]
Hi @alceil! Definitely! Please let me know if you have any questions. We also have a community Slack for chatting with other contributors and getting quick support. The link to join is https://infisical.com/slack

Thank you so much for contributing :)

**Comment by alceil** [comment]
Thanks @vmatsiiako

**Comment by alceil** [comment]
I have raised a pr to resolve this. Can you have a look? @vmatsiiako @dangtony98 @maidul98

**Comment by yadavshubham01** [comment]
@akhilmhdh i think you closed this issue it already solved
